src/test/java/gov/di_ipv_fraud/service/ConfigurationService.java

```diff
@@ -43,6 +43,7 @@ public ConfigurationService(String env) {
         this.passportCriUrl = getParameter("passportCriUrl");
         this.orchestratorStubUrl = getParameter("orchestratorStubUrl");
         this.httpsEnabled = getParameter("httpsEnabled");
+        this.fraudResultTableName = getParameter("fraudResultTableName");
```

Comment:
Hi @gds-jbentham I had to add this parameter to get my tests working on my branch that is created from the current main branch. In the stand up today, if I understood correctly you mentioned that you wanted to do this another way so that we are not passing loads of environment variables. Let me know if you would like me to remove this parameter.